examples/02_feature_interpretation_with_gapencoder.py

```diff
@@ -0,0 +1,122 @@
+"""
+.. _example_interpreting_gap_encoder:
+
+==========================================
+Feature interpretation with the GapEncoder
+==========================================
+
+In this notebook, we will explore the output and inner workings of the
+|GapEncoder|, one of the encoders provided by skrub.
+
+
+.. |GapEncoder| replace::
+     :class:`~skrub.GapEncoder`
+
+.. |SimilarityEncoder| replace::
+     :class:`~skrub.SimilarityEncoder`
+"""
+
+###############################################################################
+# The |GapEncoder| is more scalable and interpretable than the
+# |SimilarityEncoder|, which we will demonstrate now.
```

Comment:
Let's not position the GapEncoder with regards to the SimilarityEncoder, as we are no longer putting emphasis on the latter
```suggestion
# The |GapEncoder| is scalable and interpretable in terms of
# finding latent categories, as we will show.
```